src/XMakeTasks/Copy.cs

```diff
@@ -408,6 +409,20 @@ CopyFileWithState copyFile
                 _copiedFiles = new TaskItem[0];
                 return true;
             }
+            
+            if (_destinationFiles != null)
+            {
+                var sameDestinationFiles = _destinationFiles.GroupBy(x => x).Where(g => g.Count() > 1).Select(y => y.First());
```

Comment:
Should the `GroupBy` operate on the `FullPath` metadata instead? That seems more correct--but maybe I'm missing where the item's include is guaranteed to be a full path?